### PR TITLE
add ChatBox and  js for hide-show chat box

### PR DESCRIPTION
**Pull request by hapo-chinhnc**

### CODE REVIEW

File: css/home.css [hunk 1]
```diff
@@ -485,3 +489,116 @@ div.hapolearn-footer-ul a {
...
+  bottom: 0;
+  right: 0;
+}
+
+.hapolearn-messenger-chatbox {
+  width: 100%;
+  height: 190px;
+  background-color: #FFFFFF;
+  border-radius: 23px;
+  box-shadow: 0px 0px 8px rgba(0, 0, 0, 0.25);
```
Comment (hapo-lam-nv) [inline css/home.css]: 0px

File: css/home.css [hunk 2]
```diff
@@ -485,3 +489,116 @@ div.hapolearn-footer-ul a {
...
+
+.hapolearn-messenger-chatbox-login a {
+  font: normal 600 12px/16px 'Open Sans';
+  color: white;
+  background-color: #3184F7;
+  text-decoration: none;
+  padding: 10px 80px;
+  text-align: center;
+  border-radius: 10px;
+  text-align: center;
```
Comment (hapo-lam-nv) [inline css/home.css:579]: trùng

File: css/home.css (hunk 1 above)
Comment (hapo-chinhnc) [reply css/home.css]: đã fix

File: css/home.css (hunk 2 above)
Comment (hapo-chinhnc) [reply css/home.css:578]: fixed

**Pull request merged**